.github/CODEOWNERS-soft

```diff
@@ -33,14 +33,23 @@ products/web_analytics/** @PostHog/team-web-analytics
 frontend/src/scenes/web-analytics/** @PostHog/team-web-analytics
 posthog/hogql_queries/web_analytics/** @PostHog/team-web-analytics
 frontend/src/toolbar/web-vitals/** @PostHog/team-web-analytics
-posthog/hogql/database/schema/sessions_v1.py @PostHog/team-web-analytics
-posthog/hogql/database/schema/sessions_v2.py @PostHog/team-web-analytics
-posthog/models/raw_sessions/** @PostHog/team-web-analytics
-posthog/models/sessions/** @PostHog/team-web-analytics
 rust/common/cookieless/** @PostHog/team-web-analytics
 plugin-server/src/ingestion/cookieless/** @PostHog/team-web-analytics
 dags/locations/web_analytics.py @PostHog/team-web-analytics
 
+# Analytics Platform
+dags/locations/analytics_platform.py @PostHog/team-analytics-platform
+ee/tasks/subscriptions/** @PostHog/team-analytics-platform
+frontend/src/scenes/data-management/schema/** @PostHog/team-analytics-platform
+posthog/api/schema_property_group.py @PostHog/team-analytics-platform
+posthog/hogql/database/schema/sessions_*.py @PostHog/team-analytics-platform
+posthog/models/raw_sessions/** @PostHog/team-analytics-platform
```

Comment:
Can we keep the sessions bit shared? I thought you would work on v3 + sessions on events, but then move to other stuff and we would own them, but if you're keen, happy to hand them over